### ISSUE TITLE
Property enabled doesn't accept a boolean value

### ISSUE THREAD

**Issue by lafeuil** [Templarian/ui.bootstrap.contextMenu#125]
In the documentation, you specify that the property enabled of a menu can accept a boolean value and a function : 
enabled - [Function/Boolean] A function returning whether the option is enabled or not, or a boolean

But, when I define a boolean value, I obtain this stacktrace :
```
TypeError: item.enabled.call is not a function
    at isOptionEnabled (contextMenu.js:468)
    at processItem (contextMenu.js:136)
    at contextMenu.js:339
    at Object.forEach (angular.js:321)
    at renderContextMenu (contextMenu.js:318)
    at contextMenu.js:571
```

**Comment by lafeuil** [comment]
@josetaira, your fix doesn't work when item.enabled = false.
I checked the code and I have found the problem : 
At line 140 : `currItemParam.enabled = resolveBoolOrFunc(item.enabled || item[2], params);` 
If `enabled` is false, so `item[2]` is used as enabled value but it is undefined. So the true value is returned by the fonction.

I think, you must check if item.enabled is defined outside the resolveBoolOrFunc function. I propose this : 
`currItemParam.enabled = resolveBoolOrFunc(item.enabled !== undefined ? item.enabled : item[2], params);`
